### PR TITLE
Added Functionalities to Menu Button through javascript

### PR DESCRIPTION
**Pull request by Medini-Rajendra**
## In this PR, the following changes were made
 - Checked for linter errors. 
 - Readme file updated.
 - Javascript functionalities are added to html.
 - The menu button functionality has been completed as per the instructions.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+        <meta name="viewport" content="width=device-width,initial-scale=1.0">
+        <title>Website Design Project</title>
+        <link rel="stylesheet" href="css/style.css" type="text/css">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+
+    </head>
+    <body>
+        <header class="imagecontainer">
+            <div class="defineimage"></div>
+            <img src="images/ic_menu.svg" alt="menu" onclick="mobile_menu()">
```
Comment (Shaher-11) [inline index.html:14]: the x sign for closing the menu is not visible 
### Original design
right

### Your Project
wrong

File: index.html [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+        </header>
+        <div class="horizontal_line">
+            <a href="#text1" class="wrap ite1">Hello</a>
+            <a href="#text3" class="wrap ite2">Portfolio</a>
+            <a href="#abouttext" class="wrap ite4">About</a>
+            <a href="#text10" class="wrap ite5">Contact</a>
+        </div>
```
Comment (Shaher-11) [inline index.html:21]: Kindly use the ul tag and style the items as shown in the original design below
### Original design
right

### Your Project
wrong

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @Medini-Rajendra,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Check the comments below:point_down:


>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

File: index.html (hunk 1 above)
Comment (Medini-Rajendra) [reply index.html:14]: I have made the changes as per your suggestion. Please view my code using responsive 400 in mobile view.

File: index.html (hunk 2 above)
Comment (Medini-Rajendra) [reply index.html:21]: I have made the changes as per the reviewer's suggestions. The shown picture here is the menu design coded in javascript. And the ul, li elements are used in javascript coding. Kindly refer to my javascript code. I have finished the changes and updated the code.

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**